LibraBFT/Impl/Consensus/BlockStorage/BlockStore.agda

```diff
@@ -142,8 +142,13 @@ executeAndInsertBlockE bs0 block =
     else do
       eb ← case executeBlockE bs0 block of λ where
         (Right res) → Right res
-        (Left (ErrBlockNotFound parentBlockId)) → do
+        -- OBM-LBFT-DIFF : This is never thrown in OBM.
+        -- Because it is thrown by StateComputer in Rust (but not in OBM).
+        (Left (ErrECCBlockNotFound parentBlockId)) → do
           eitherS (pathFromRoot parentBlockId bs0) Left $ λ blocksToReexecute →
+            -- OBM-LBFT-DIFF : OBM StateComputer does NOT have state.
+            -- If it ever does have state then the following 'forM' will
+            -- need to change to some sort of 'fold'.
```

Comment:
I would add something like: "because `executeBlockE` would change the state, so the state passed to `executeBlockE` in the `Right` case below" would no longer be `bs0`.

Reply:
DONE.

Reply:
I updated the PR description to describe the other changes.